pkg/cloudprovider/aws/apis/v1alpha1/register.go

```diff
@@ -58,7 +58,10 @@ var (
 	ResourceAWSPodENI v1.ResourceName = "vpc.amazonaws.com/pod-eni"
 
 	LabelInstanceHypervisor      = LabelDomain + "/instance-hypervisor"
+	LabelInstanceCategory        = LabelDomain + "/instance-category"
 	LabelInstanceFamily          = LabelDomain + "/instance-family"
+	LabelInstanceGeneration      = LabelDomain + "/instance-generation"
+	LabelInstanceLocalNVME       = LabelDomain + "/instance-local-nvme"
```

Comment:
From offline discussion where you called these capabilities, should this be `instance-capability-local-nvme`?